Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -473,28 +458,29 @@ namespace OpenInfraPlatform {
 					polylineData->addPolylineIndex(2);
 					polylineData->addPolylineIndex(3);
 
+					return polylineData;
 					throw oip::UnhandledException(surface);

```

Comment:
Why does this function throw, if the entity is being handled?

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/7ac16ab3b8ddb7adc9c410d2aafc5efa07a39aca